tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Get Calculations.feature

```diff
@@ -0,0 +1,36 @@
+﻿Feature: Get the price calculation details of an order
+    As an Buyer user
+    I want to be able to view the pricing of a given order
+    So that I can ensure that the information is correct
+
+Background:
+    Given Addresses exist
+        | Line1 | Line2      | Line3      | Line4          | Line5          | Town      | County  | Postcode | Country        |
+        | 4     | Upper Flat | Rocks Lane | Little Village | Bigger Village | Some Town | W Yorks | LS 1 3AP | United Kingdom |
+    Given Contacts exist
+        | FirstName | LastName | EmailAddress            | TelephoneNumber |
+        | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
+    Given Orders exist
+        | OrderId    | Description   | OrganisationId                       | OrganisationName | OrganisationOdsCode | OrganisationContactEmail | SupplierAddressPostcode | SupplierContactEmail    |
+        | C000014-01 | A Description | 4af62b99-638c-4247-875e-965239cd0c48 | Hampshire CC     | 432432              | Fred.robinson@email.com  | LS 1 3AP                | Fred.robinson@email.com |
+    Given Service Recipients exist
+        | OrderId    | OdsCode | Name    |
+        | C000014-01 | eu      | EU Test |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+        
+Scenario: 1. Blah
```

Comment:
Blah?

Reply:
It's an acronym, meaning Business Logic Appears Healthy... updating to something more human readable!